src/Krank/Formatter.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Krank.Formatter (
+  showViolations
+  ) where
+
+import Data.Text (Text, append, pack)
+
+import Krank.Types
+
+showViolations :: [Violation]
+               -> Text
+showViolations = foldl (\acc v -> acc `append` showViolation v) ""
+
+showViolation :: Violation
+              -> Text
+showViolation violation = "\n"
```

Comment:
I'd like to use `PyF` for this kind of formatting.

Reply:
sounds good to me. I'll look into it